app_dart/bin/server.dart

```diff
@@ -82,29 +93,135 @@ Future<void> main() async {
         authProvider,
         scheduler,
       ),
+
+      /// Updates cocoon task status.
```

Comment:
I would call this a holistic task update as it also updates performance data

Reply:
changed